src/Documents/Operations/Etl/RavenEtlConfiguration.ts

```diff
@@ -0,0 +1,16 @@
+import { EtlConfiguration } from "./EtlConfiguration";
+import { DocumentConventions, EtlType, RavenConnectionString } from "../../..";
```

Comment:
This smells. As a rule of a thumb we do not use main src/index for internal imports.

Reply:
this is fine.